04_HTML/Praktikum/CreateAccount.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Create Account</title>
+</head>
+
+<body>
+    <form>
+        <h1>Create Account</h1>
+        <h2>Detail Form</h2>
+
+        <p>First name :</p>
+        <input type="text" required>
+
+        <p>Last name :</p>
```

Comment:
sama seperti komen di atas, semua p yg bertujuan sebagai label input harus menggunakan tag label dg for attribute